tsl/test/t/003_connections.pl

```diff
@@ -66,3 +66,75 @@
 is( $stdout,
 	q[2021-09-19 18:00:00|1|23.4
 dn1|alice|f]);
+
+# Test DROP OWNED BY and REASSIGN OWNED BY behavior across the setup
+# Create a bunch of roles and use them to set up grant dependencies and
+# object ownership
+$an->safe_psql(
+	'postgres',
+	qq[
+CREATE USER regress_dep_user0;
+CREATE USER regress_dep_user1;
+CREATE USER regress_dep_user2;
+call distributed_exec(\$\$ CREATE USER regress_dep_user0; \$\$);
+call distributed_exec(\$\$  CREATE USER regress_dep_user1; \$\$);
+call distributed_exec(\$\$ CREATE USER regress_dep_user2;\$\$);
+GRANT USAGE ON FOREIGN SERVER dn1,dn2 TO regress_dep_user0;
+GRANT USAGE ON FOREIGN SERVER dn1,dn2 TO regress_dep_user1;
+GRANT USAGE ON FOREIGN SERVER dn1,dn2 TO regress_dep_user2;
+SET SESSION AUTHORIZATION regress_dep_user0;
+CREATE TABLE conditions1 (time bigint NOT NULL, device int, temp float);
+SELECT create_distributed_hypertable('conditions1', 'time', chunk_time_interval => 10);
+GRANT ALL ON conditions1 TO regress_dep_user1 WITH GRANT OPTION;
+SET SESSION AUTHORIZATION regress_dep_user1;
+GRANT ALL ON conditions1 TO regress_dep_user2 WITH GRANT OPTION;
+    ]);
+
+# Check the access privileges an AN and DN
+my $query =
+  q[SELECT pg_catalog.array_to_string(c.relacl, E'\n') FROM pg_catalog.pg_class c WHERE c.relname = 'conditions1';];
+
+$an->psql_is(
```

Comment:
While a TAP test is good, I am wondering if this can't more easily be tested in a regular regression test, e.g., `dist_grant`?

Reply:
The main reason was that I didn't want to update 3 versions of the output files for `dist_grant` :-)